pkgs/development/compilers/rust/default.nix

```diff
@@ -26,10 +26,7 @@ in rec {
     # 2. Tests run out of memory for i686
     doCheck = !stdenv.isAarch64 && !stdenv.isi686;
 
-    patches = [
-      ./patches/0001-Disable-fragile-tests-libstd-net-tcp-on-Darwin-Linux.patch
```

Comment:
Why was it removed?

Reply:
Well, the sources changed. Not sure how to fix them.